lib/go/test/token_test.go

```diff
@@ -3,7 +3,8 @@ package test
 import (
 	"testing"
 
-	emulator "github.com/dapperlabs/flow-emulator"
+	emulator "github.com/onflow/flow-emulator"
+	templates2 "github.com/onflow/flow-go-sdk/templates"
```

Comment:
```suggestion
	sdktemplates "github.com/onflow/flow-go-sdk/templates"
```

Maybe a more descriptive name here?

Reply:
Good idea, improved it in 741ffbe6e5cffb266b01063f32fc23d423a928ed